packages/p/pffft/xmake.lua

```diff
@@ -0,0 +1,25 @@
+package("pffft")
+
+    set_homepage("https://bitbucket.org/jpommier/pffft/")
+    set_description("PFFFT, a pretty fast Fourier Transform.")
+    set_license("BSD-like (FFTPACK license)")
+
+    add_urls("https://bitbucket.org/jpommier/pffft/$(version).tar.gz",
+             "https://bitbucket.org/jpommier/pffft.git")
+    add_versions("02fe7715a5bf8bfd914681c53429600f94e0f536", "9adeb18ac7bb52e9fb921c31c0c6a4e9ae150cc6fcb20a899d4b3a2275176ded")
```

Comment:
only support semver, you need map it to commit in version function. see other packages. e.g. sqlite3

and add a new line

Reply:
https://github.com/microsoft/vcpkg/blob/80d54ff62d528339c626a6fbc3489a7f25956ade/ports/pffft/portfile.cmake#L6-L8
https://github.com/conan-io/conan-center-index/blob/4622ac85d1cec8cb7a2fcc8a1796d4b73bff285e/recipes/pffft/all/conandata.yml#L1-L7
We could go back into 
```lua
add_urls("https://bitbucket.org/jpommier/pffft/get/$(version).zip")
add_versions("02fe7715a5bf", "81e1b91bad77092681e5ed6c2eef1a3bae7eb2154d3358a10c663867cd947396")
```